### PR TITLE
First draft of IPFS Weekly #52

### PR DESCRIPTION
**Pull request by jennwrites**
Looking for feedback and suggestions on this draft by **Monday, July 29 at 12pm Central** so that we can send it out via TinyLetter (and post on the blog) Tuesday, July 30. 🏝

Thanks!

### CODE REVIEW

**Comment by ianopolous** [comment]
Is there planned to be a section about new releases? If so, could we please include:

A new release of Peergos: v0.0.11 (https://alpha.peergos.net/public/peergos/releases/v0.0.11), with qr code friend verification, folder upload and more!

**Comment by terichadbourne** [comment]
Please go ahead and remove the Offline Camp mention moving forward since we won’t have time to accept more applications before the event. I’ll keep you posted on resulting blog posts and such after the event. Thanks!

File: published/052-2019-july-30.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
++ [Unbundling Vectors of Centralization in Web3](https://multicoin.capital/2019/07/25/unbundling-vectors-of-centralization-in-web3/), Multicoin Capital, 25 July 2019
++ Check out [Elk: The Dev Board for the Decentralized World](https://www.kickstarter.com/projects/233173198/elk-the-dev-board-for-the-decentralized-world), Kickstarter, 25 July 2019
++ [Report: A study of libp2p and ETH2](https://discuss.libp2p.io/t/report-a-study-of-libp2p-and-eth2/229), Raúl Kripalani, 24 July 2019
++ [Re-decentralizing the Web](https://medium.com/@hajanetworks/re-decentralizing-the-web-54678a1e4848), Haja Networks, 24 July 2019
++ [Smart Contract Based IPFS Storage for DAOs](https://medium.com/open-work-labs/smart-contract-based-ipfs-storage-for-daos-39c145f3042d), Open Work Labs, 24 July 2019
++ [ARK Core: Path And Vision To V3](https://blog.ark.io/ark-core-path-and-vision-to-v3-7a8bc3338d5a), ARK.io, 24 July 2019
++ [10+ Resources Every IPFS Developer Should Know About](https://medium.com/swlh/10-resources-to-get-started-with-ipfs-5f429dc8a841), The Startup, 23 July 2019
++ [IPFS Camp, new features, July 2019](https://peergos.org/blog#ipfs_camp_new_features_july_2019_), Peergos, 22 July 2019
```
Comment (ianopolous) [inline published/052-2019-july-30.md]: Could we please change the link title here to "Peergos at IPFS Camp, July 2019" please? Thank you.

**Comment by jennwrites** [comment]
IPFS Weekly #52 was published

- On the [IPFS blog](still pending)
- To 2868 subscribers via [Tinyletter](https://tinyletter.com/ipfsnewsletter/letters/ipfs-weekly-52)

**Comment by Leela-ann** [comment]
Hi @renrutnnej where should I post the upcoming IPFS Meet & Hacks taking place monthly in SF?

**Comment by jennwrites** [comment]
> Hi @renrutnnej where should I post the upcoming IPFS Meet & Hacks taking place monthly in SF?

@Leela-ann On Thursdays I open a new PR for the upcoming week’s draft so you can add info there, or you can email newsletter@ipfs.io the info, and please include a link! Thanks ☺️

**Comment by Leela-ann** [comment]
@renrutnnej got it - thanks!

**Pull request merged**